futures-util/src/lock/mutex.rs

```diff
@@ -381,19 +514,35 @@ impl<T: ?Sized, U: ?Sized> DerefMut for MappedMutexGuard<'_, T, U> {
 // Mutexes can be moved freely between threads and acquired on any thread so long
 // as the inner value can be safely sent between threads.
 unsafe impl<T: ?Sized + Send> Send for Mutex<T> {}
+
```

Comment:
(I personally like the previous format where single line trait implementations for the same type are grouped, but I'm not opposed to this change.)

Reply:
I have reverted the format, the format of the codes looks the same as the previous version